block.go

```diff
@@ -242,6 +242,27 @@ type Quote struct {
 	Children Blocks     `json:"children,omitempty"`
 }
 
+type TableBlock struct {
+	BasicBlock
+	Table Table `json:"table"`
+}
+
+type Table struct {
+	TableWidth      int    `json:"table_width"`
+	HasColumnHeader bool   `json:"has_column_header"`
+	HasRowHeader    bool   `json:"has_row_header"`
+	Children        Blocks `json:"children,omitempty"`
```

Comment:
There is no information about this field in the documentation. Do we really need `Children` here?

Reply:
I think the documentation lose it. Because we need to save table_row by the `Children `

Reply:
Like the Paragraph, Callout...